src/tools/goid-tool.c

```diff
@@ -855,23 +855,21 @@ main(int argc, char **argv)
     memset(&ctx_param, 0, sizeof(ctx_param));
     ctx_param.ver      = 0;
     ctx_param.app_name = app_name;
+	ctx_param.debug    = cmdline.verbose_given;
+    if (cmdline.verbose_given > 1)
+		ctx_param.debug_file = stderr;
```

Comment:
there is some mixture of tabs and spaces for indentation.

Reply:
hmm, this file uses spaces only. I'll change the PR to stick to this.